magefile.publish.go

```diff
@@ -29,19 +29,17 @@ type Distro struct {
 }
 
 var targetDistros = []Distro{
-	{OS: "el", Dist: "6"},
 	{OS: "el", Dist: "7"},
 	{OS: "el", Dist: "8"},
 	{OS: "fedora", Dist: "29"},
 	{OS: "fedora", Dist: "30"},
```

Comment:
Shouldn't we add Fedora 31-35?

Reply:
Dropped 29, added 31-35

Reply:
What about "opensuse"? I suggest adding them as well.

Reply:
curl flags (excluding secrets): [-v -LfsS -X PUT https://test-tarantool-rws.herokuapp.com/release/modules/fedora/35 -F product=cartridge-cli -F cartridge-cli-2.10.0.x86_64.rpm=@./dist/cartridge-cli-2.10.0.x86_64.rpm]
Error: Failed to publish package for fedora/35: exit status 22, *   Trying 46.137.15.86:443...
* TCP_NODELAY set
* Connected to test-tarantool-rws.herokuapp.com (46.137.15.86) port 443 (#0)
* ALPN, offering h2
* ALPN, offering http/1.1
* successfully set certificate verify locations:
*   CAfile: /etc/ssl/certs/ca-certificates.crt
  CApath: /etc/ssl/certs
} [5 bytes data]
* TLSv1.3 (OUT), TLS handshake, Client hello (1):
} [512 bytes data]
* TLSv1.3 (IN), TLS handshake, Server hello (2):
{ [91 bytes data]
* TLSv1.2 (IN), TLS handshake, Certificate (11):
{ [4935 bytes data]
* TLSv1.2 (IN), TLS handshake, Server key exchange (12):
{ [333 bytes data]
* TLSv1.2 (IN), TLS handshake, Server finished (14):
{ [4 bytes data]
* TLSv1.2 (OUT), TLS handshake, Client key exchange (16):
} [70 bytes data]
* TLSv1.2 (OUT), TLS change cipher, Change cipher spec (1):
} [1 bytes data]
* TLSv1.2 (OUT), TLS handshake, Finished (20):
} [16 bytes data]
* TLSv1.2 (IN), TLS handshake, Finished (20):
{ [16 bytes data]
* SSL connection using TLSv1.2 / ECDHE-RSA-AES128-GCM-SHA256
* ALPN, server did not agree to a protocol
* Server certificate:
*  subject: CN=*.herokuapp.com
*  start date: Jun  1 00:00:00 2021 GMT
*  expire date: Jun 30 23:59:59 2022 GMT
*  subjectAltName: host "test-tarantool-rws.herokuapp.com" matched cert's "*.herokuapp.com"
*  issuer: C=US; O=Amazon; OU=Server CA 1B; CN=Amazon
*  SSL certificate verify ok.
* Server auth using Basic with user 'rwscd'
} [5 bytes data]
> PUT /release/modules/fedora/35 HTTP/1.1
> Host: test-tarantool-rws.herokuapp.com
> Authorization: Basic cndzY2Q6WXRySHJ4eGNZdXphczEzMzcxMzM4TG9M
> User-Agent: curl/7.68.0
> Accept: */*
> Content-Length: 4662855
> Content-Type: multipart/form-data; boundary=------------------------a539da969b6aebff
> Expect: 100-continue
> 
{ [5 bytes data]
* Mark bundle as not supporting multiuse
< HTTP/1.1 100 Continue
} [5 bytes data]
* We are completely uploaded and fine
{ [5 bytes data]
* Mark bundle as not supporting multiuse
* The requested URL returned error: 400 BAD REQUEST
* Closing connection 0
} [5 bytes data]
* TLSv1.2 (OUT), TLS alert, close notify (256):
} [2 bytes data]
curl: (22) The requested URL returned error: 400 BAD REQUEST